proto/executor/v1/executor.proto

```diff
@@ -45,6 +45,13 @@ message ProcessBatchResponse {
     uint64 cumulative_gas_used = 12;
     repeated ProcessTransactionResponse responses = 13;
     Error error = 14;
+    repeated TouchedAddresses touched_addresses = 15;
```

Comment:
```suggestion
    map[string]TouchedAddresses touched_addresses = 15;
```

Reply:
It will be better if we make this mapping for quick lookup by address (we can use string or byte[] for the key).